src/app/component.scss

```diff
@@ -0,0 +1,90 @@
+@mixin column_styling($prop1, $prop2) {
+  display: flex;
+  flex-flow: column;
+  justify-content: $prop1;
+  text-align: center;
+  margin-top: $prop2;
+}
+
+@mixin skill-icon($prop1, $prop2, $prop3) {
+  background-color: $prop1;
+  font-size: $prop2;
+  opacity: $prop3;
+}
+
+@mixin button-element($prop1) {
+  margin-top: $prop1;
+  width: 200px;
+  height: 100px;
+  border: none;
+  &:hover {
+    width: 250px;
+    height: 150px;
+    & .hover-tip-note {
```

Comment:
This is quite confusing that this part of css is distinct from basic hover-tip-note.

Also does it need to be inside button-element? Isn't it just the property of this class?